oracle/README.md

```diff
@@ -52,37 +52,14 @@ When you run `datadog-agent info` you should see something like the following:
 
 ### Compatibility
 
-The oracle check is currently compatible with the linux and darwin-based OS
+The Oracle check is currently compatible with Linux and macOS.
 
 ## Data Collected
 ### Metrics
 See [metadata.csv](https://github.com/DataDog/integrations-core/blob/master/oracle/metadata.csv) for a list of metrics provided by this integration.
 
 ### Events
-The oracle check does not include any event at this time.
+The Oracle Database check does not include any event at this time.
```

Comment:
@marcweisman @l0k0ms @truthbk @handigarde Here we say this integration collects no events, but on line 8 we say it has events.

Reply:
This is correct.